src/css/index.css

```diff
@@ -0,0 +1,61 @@
+* {
+	font-family: Helvetica, sans-serif;
+	box-sizing: border-box;
+	padding: 0;
+}
+
+body {
+	margin: 0;
+}
+
+.list-navigation {
+	height: 80px;
+	background-color: #6b5b95;
+	display: flex;
+	align-items: flex-end;
+}
+
+.list-navigation__search-wrapper {
+	display: flex;
+	align-items: flex-end;
+	justify-content: flex-end;
+	border-radius: 8px;
+	background-color: #f5f5f5;
+	height: 30px;
+	width: 300px;
+	margin: 5px;
+}
+
+.list-navigation__search-input {
+	width: 300px;
+	height: 30px;
+	border-radius: 8px 0px 0px 8px;
+	background-color: #f5f5f5;
+	border: 1px solid grey;
+	border-right: none;
+	outline: 0;
+	font-size: 18px;
+	padding-left: 5px;
+}
+.list-navigation__search-input:hover,
+.list-navigation__search-input:focus {
+	border: 1px solid #009688;
+	background-color: white;
+}
+
+.list-navigation__search-button:hover {
+	border-color: #009688;
+	background-color: white;
+}
+
+.list-navigation__search-button {
+	width: 50px;
+	height: 30px;
+	background-color: #f5f5f5;
+	border: 1px solid grey;
+	outline: none;
+	cursor: pointer;
```

Comment:
This is redundant because button have cursor pointer by default

Reply:
i've noticed that it has a cursor pointer, i wanted a hand pointer specifically.